mm2src/coins/nft/storage/wasm/wasm_storage.rs

```diff
@@ -101,6 +130,14 @@ impl NftTransferHistoryFilters {
         self.from_date.map_or(true, |from| transfer.block_timestamp >= from)
             && self.to_date.map_or(true, |to| transfer.block_timestamp <= to)
     }
+
+    fn is_spam_match(&self, transfer: &NftTransferHistory) -> bool {
+        !self.exclude_spam || !transfer.common.possible_spam
+    }
+
+    fn is_phishing_match(&self, transfer: &NftTransferHistory) -> bool {
+        !self.exclude_phishing || !transfer.possible_phishing
+    }
```

Comment:
Probably different fn names would be better here as non of these functions doesn't make any comparison but the name includes `match`

Reply:
oh, agree, what about `passes_spam_filter` and `passes_phishing_filter`?

Reply:
Looks good!

Reply:
done